docs/topics/readme/rest-http-secured-README.adoc

```diff
@@ -26,6 +26,7 @@ $ oc create -f service.sso.yaml
 $ ${localRunCMD}
 ----
 
+${integrationTestSection}
```

Comment:
For the Secured boosters, this variable is not defined. Also it's unfortunately not as simple as for the other boosters, so I'd rather comment this out for now.

(There's unfortunately a difference between boosters here. The Vert.x integration tests deploy the SSO server as part of the test, while Swarm and Spring Boot tests don't.)

Reply:
This commet is still valid.

Also, should probably add the same info to https://github.com/fabric8-launcher/launcher-documentation/pull/1061 or plan to do the same after.

Reply:
@Ladicek Thank you. Given the relatively subtle but important differences in the way integration tests are executed against the Secured booster on different runtimes, I propose we include a link to the integration test section for the respective booster rather than citing specific steps and/or command examples in the README.

Reply:
Works for me, thanks!